lib/helpers/index.js

```diff
@@ -0,0 +1,59 @@
+exports = module.exports = {
+    /**
+     * Validate that response header exists
+     * @private
+     * @param {object} request
+     */
+    hasHeaders: function (request) {
+        let value = false;
+
+        if (typeof request.response === 'object' &&
+            request.response !== null &&
+            typeof request.response.header === 'function') {
+            value = true;
+        }
+
+        return value;
+    },
```

Comment:
If that's true this can be removed since we are controlling the cnn-hapi version ... perhaps out of scope of this PR, hazards of touching a lot of lines :-P